yarn.lock

```diff
@@ -5359,11 +5359,6 @@ clone-response@^1.0.2:
   dependencies:
     mimic-response "^1.0.0"
 
-clone@2.x:
```

Comment:
Do we need to upgrade yarn.lock in this PR?

Reply:
No, looks like my yarn's out of date, which causes this change, I'll revert this and update yarn locally. Thanks for spotting this!